### PR TITLE
[ALLUXIO-2257] Improve naming in DynamicResourcePoolTest

### PR DESCRIPTION
**Pull request by ayuileng**
https://alluxio.atlassian.net/browse/ALLUXIO-2257

### CODE REVIEW

**Comment by ayuileng** [comment]
UnHealhyResource should be UnhealthyResource

File: core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java [hunk 1]
```diff
@@ -144,7 +144,7 @@ public void acquireWithoutCapacity() throws Exception {
    * Tests the logic that invalid resource won't be acquired.
    */
   @Test
-  public void UnHealhyResource() throws Exception {
+  public void UnhealhyResource() throws Exception {
```
Comment (gpang) [inline core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java]: This should be rename to "UnhealthyResource"

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

**Comment by ayuileng** [comment]
I have improved my changes

**Pull request merged**